From 5e4a2af3c2298654d420fbae30b14ab5f568b427 Mon Sep 17 00:00:00 2001
From: Jean-Christophe Morin <jcmorin@anaconda.com>
Date: Mon, 18 Dec 2023 13:17:01 -0500
Subject: [PATCH] Adjust timeseries requirements in setup.py

Deviating slightly from upstram. Upstream wants <1.5,>=1.4.0. For now we have 1.5 on sfe1ed40. 1.5 should be fine.
---
 timeseries/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/timeseries/setup.py b/timeseries/setup.py
index 076069f6..e88cff08 100644
--- a/timeseries/setup.py
+++ b/timeseries/setup.py
@@ -32,7 +32,7 @@ install_requires = [
     "gluonts>=0.14.0,<0.15",
     "networkx",  # version range defined in `core/_setup_utils.py`
     # TODO: update statsforecast to v1.5.0 - resolve antlr4-python3-runtime dependency clash with multimodal
-    "statsforecast>=1.4.0,<1.5",
+    "statsforecast>=1.4.0,<1.6",
     "mlforecast>=0.10.0,<0.10.1",
     "utilsforecast>=0.0.10,<0.0.11",
     "tqdm",  # version range defined in `core/_setup_utils.py`
-- 
2.43.0

